core/blockchain.go

```diff
@@ -2541,15 +2541,21 @@ func (bc *BlockChain) reportBlock(block *types.Block, res *ProcessResult, err er
 
 // logForkReadiness will write a log when a future fork is scheduled, but not
 // active. This is useful so operators know their client is ready for the fork.
-func (bc *BlockChain) logForkReadiness(block *types.Block) {
+func (bc *BlockChain) logForkReadiness() {
 	c := bc.Config()
-	current, last := c.LatestFork(block.Time()), c.LatestFork(math.MaxUint64)
+	now := time.Now()
+	current, last := c.LatestFork(uint64(now.Unix())), c.LatestFork(math.MaxUint64)
 	t := c.Timestamp(last)
 	if t == nil {
 		return
 	}
 	at := time.Unix(int64(*t), 0)
-	if current < last && time.Now().After(bc.lastForkReadyAlert.Add(forkReadyInterval)) {
```

Comment:
it confuses me a lot that:

in theory, once we pass the prague fork, the current and last should all refer to PragueFork and `current < last` is not true, right?